### PR TITLE
Add Lingvo test runner to CI

### LINKED ISSUE #707: Add test runner for Lingvo ASR estimator
The Lingvo ASR estimator requires a very specific combination of `lingvo` and `tensorflow-gpu` that is not supported on CI. Therefore, CI will skip those tests at the moment.

Given the work of @ebubae, who added Github Actions to our CI, we can now easily add new test runners. I would like to propose that we add a specific job for the Lingvo estimator.

### PR DESCRIPTION
**Pull request by hesseltuinhof**
# Description

This PR adds a Lingvo test runner to the CI pipeline. All tests related to Lingvo, require a specific version of `tensorflow` (`2.1.0`), Lingvo (`0.6.4`) and Python 3.6.

Note, the new test runner will run all `tensorflow2v1` flagged tests.

Fixes #707

## Type of change

Please check all relevant options.

- [x] Improvement (non-breaking)

**Test Configuration**:

- Fedora 33
- Python 3.6

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -79,6 +79,14 @@ jobs:
           - name: mxnet (Python 3.7)
             framework: mxnet
             python: 3.7
+          - name: TensorFlow+Lingvo 2.1.0v1 (Keras 2.3.1 Python 3.6)
+            framework: tensorflow2v1
+            python: 3.6
+            tensorflow: 2.1.0
+            tenorflow_gpu: 2.1.0
```
Comment (beat-buesser) [inline .github/workflows/ci.yml:86]: Is the label `tensorflow_gpu` still required? I think it's covered by label `tensorflow`.

File: setup.cfg [hunk 2]
```diff
@@ -9,6 +9,7 @@ markers =
     only_with_platform: DEPRECATED only used for legacy tests. Use skipMlFramework instead. marks a test to be performed only for a specific mlFramework values
     framework_agnostic: marks a test to be agnostic to frameworks and run only for one default framework
     skip_travis: Skips a test marked with this decorator if the command line argument skip_travis is set to true
+    skipModule: Skip the test if a module is not available in the current environment
```
Comment (beat-buesser) [inline setup.cfg:12]: How about adapting the marker name following the existing naming pattern? Although `skipMlFramework` is also an outlier.
```suggestion
    skip_module: Skip the test if a module is not available in the current environment
```

**Review by beat-buesser — CHANGES_REQUESTED** [review]
Hi @hesseltuinhof Thank you very much for your pull request! I like the changes and have a few question which might add additional improvements. What do you think?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (hesseltuinhof) [reply .github/workflows/ci.yml:86]: Yes, you are right. 

I also checked the `setup.py` of the Lingvo library and they install `tensorflow-gpu` via `tensorflow-gpu==tf.__version__` (see [setup.py#L43](https://github.com/tensorflow/lingvo/blob/55263937599fdbbfb959e8eab7a40fa0aa74ad9c/pip_package/setup.py#L43)). So we can stick to the same snippets we use for the other Tensorflow workflows.

File: setup.cfg (hunk 2 above)
Comment (hesseltuinhof) [reply setup.cfg]: The original code to skip modules uses camel case. I agree that both `skipModule` and `skipMlFramework` are outliers. So let's start to fix the `skip_module` with this PR :relaxed:

I will open an issue for `skipMlFramework` and if time allows update it as well :+1:

**Comment by hesseltuinhof** [comment]
Thanks for suggesting the two changes. I fully agree with them and have applied the PR accordingly :+1:

**Review by beat-buesser — APPROVED** [review]
@hesseltuinhof Thank you!

**Pull request merged**